src/graphql/types/object/btc-wallet.ts

```diff
@@ -43,6 +43,15 @@ const BtcWallet = GT.Object<Wallet>({
         return balanceSats
       },
     },
+    pendingIncomingBalance: {
+      type: GT.NonNull(SignedAmount),
+      description: "An unconfirmed incoming onchain balance stored in BTC.",
```

Comment:
```suggestion
      description: "An unconfirmed incoming onchain balance.",
```

`stored in BTC` feels redundant? we are in BTC wallet.

Plus I believe the balance in in sats so the `stored in BTC` can be confusing.